tests/release/pipelines/push_to_external_registry.go

```diff
@@ -164,12 +163,12 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 		})
 		Expect(err).NotTo(HaveOccurred())
 
-		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlanAdmission(releaseConst.TargetReleasePlanAdmissionName, managedNamespace, releaseConst.ReleaseEnvironment, devNamespace, releaseConst.ReleaseStrategyPolicyDefault, releaseConst.ReleasePipelineServiceAccountDefault, []string{releaseConst.ApplicationNameDefault}, true, &tektonutils.PipelineRef{
+		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlanAdmission(releaseConst.TargetReleasePlanAdmissionName, managedNamespace, "", devNamespace, releaseConst.ReleaseStrategyPolicyDefault, releaseConst.ReleasePipelineServiceAccountDefault, []string{releaseConst.ApplicationNameDefault}, true, &tektonutils.PipelineRef{
 			Resolver: "git",
 			Params: []tektonutils.Param{
 				{Name: "url", Value: releaseConst.RelSvcCatalogURL},
 				{Name: "revision", Value: releaseConst.RelSvcCatalogRevision},
-				{Name: "pathInRepo", Value: "pipelines/push-to-external-registry/push-to-external-registry.yaml"},
+				{Name: "pathInRepo", Value: "pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml"},
```

Comment:
Can you rename this file accordingly too?

Reply:
In development/staging/production branches,   the "rh-push-to-external-registry" pipeline are needed. Do we need to consider "main" branch?

Reply:
No, we don't need to consider the main branch. I meant this file is called `tests/release/pipelines/push_to_external_registry.go` can you make it `tests/release/pipelines/rh_push_to_external_registry.go`?

Reply:
Updated, thanks!